format/format.go

```diff
@@ -0,0 +1,45 @@
+// Package format defines the format of a Litmus test file, as represented in code.
+package format
+
+// LitmusFile is the top level container element defining a Litmus test file
+type LitmusFile struct {
```

Comment:
How's about TestFile instead of LitmusFile?  We're already in the context of the litmus application, so it's a little stuttery, even if it's not actual stuttering :P

Reply:
fixed